daffodil-cli/src/test/scala/org/apache/daffodil/cli/cliTest/TestCLIParsing.scala

```diff
@@ -669,6 +669,24 @@ class TestCLIParsing {
     }(ExitCode.Success)
   }
 
+  @Test def test_DFDL_1203_schema_from_absolute_path(): Unit = {
+    // This test makes sure a schema can be compiled from an absolute path on the filesystem
+    val schema = path(
+      "daffodil-cli/src/test/resources/org/apache/daffodil/cli/global_element.dfdl.xsd",
+    ).toAbsolutePath
```

Comment:
Is this going to be a MS-Windows style path when this test is run on Windows?
I honestly did not know that. I assumed Java paths were somehow portable things at that point. 
If so, please add comment like // on MS Windows this will be a windows-style path with backslash separators.

Reply:
Yes, the `path` method just does `Paths.get(...)` which supports unix-style separators even on windows systems. And when call `toString` on that `Path` to get a string to pass in to the CLI, the string contains windows style paths.

So `Paths.get("/path/containing/unix/separators").toString` will work on Linux or Windows, and return the right thing depending on the OS.